Source/Core/DolphinQt/QtUtils/VerticalTabsTabWidget.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "DolphinQt/QtUtils/VerticalTabsTabWidget.h"
+
+#include <QStyleOptionTab>
+#include <QStylePainter>
+
+namespace
+{
+class HorizontalTabBar : public QTabBar
+{
+public:
+  explicit HorizontalTabBar(QWidget* const parent = nullptr) : QTabBar(parent)
+  {
+    setUsesScrollButtons(false);
+  }
+
+protected:
+  QSize tabSizeHint(const int index) const override
+  {
+    return QTabBar::tabSizeHint(index).transposed();
+  }
+
+  void paintEvent(QPaintEvent* const event) override
+  {
+    QStylePainter painter(this);
+
+    const int current_tab_index{currentIndex()};
+
+    const int tab_count{count()};
+    for (int i{0}; i < tab_count; ++i)
+    {
+      if (i != current_tab_index)
+        paintTab(painter, i);
+    }
+
+    // Current tab is painted last as, depending on the [system] style, it is possible that the
+    // decoration is required to occlude the adjacent tab underneath.
+    paintTab(painter, current_tab_index);
+  }
+
+private:
+  void paintTab(QStylePainter& painter, const int tab_index)
```

Comment:
```suggestion
  void PaintTab(QStylePainter& painter, const int tab_index)
```
PascalCase

Reply:
Changed to `PaintTab()` now.

(For some reason, I always thought it was camelCase for _method_ names and PascalCase for top-level or class function names, and snake_case for variables and method/function parameter names.)